.goreleaser.yaml

```diff
@@ -0,0 +1,66 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  - main: ./cmd/celestia-appd
+    binary: celestia-appd
+    env:
+      - CGO_ENABLED=0
+      - SDKPath={{ "github.com/cosmos/cosmos-sdk/version" }}
+    goos:
+      - linux
+      - windows
+      - darwin
+    flags:
+      - -tags={{ .Env.BuildTags }}
```

Comment:
I honestly would prefer the build flags to be listed here explicitly too:
https://github.com/cosmos/gaia/blob/c54975c3eb9e146f127e0870e4305ae7de605635/.goreleaser.yml#L12C5-L14

Reply:
honestly that would make this way cleaner haha 

is it always just adding `ledger`? I wasn't totally sure what that build tags script was checking for so I just tried to mimic the full functionality.

Reply:
based on [this comment](https://github.com/celestiaorg/celestia-app/pull/2437#discussion_r1317426302) I'm going to resolve this as I don't want to change any build behavior in this PR, just referencing the existing code.